lua/terrortown/lang/en.lua

```diff
@@ -1864,6 +1864,7 @@ L.tbut_adminarea = "ADMIN AREA:"
 -- 2023-08-10
```

Comment:
Could you do us a favor and add those too to all other languages?

Reply:
i didn't understand at first, but I think I got it now